common/predictive-text/package.json

```diff
@@ -3,10 +3,13 @@
   "version": "0.20181001.0",
   "description": "Keyman/Predictive Text integration layer",
   "scripts": {
-    "test": "bash ./unit_tests/test.sh",
+    "build-main": "tsc -p ./tsconfig.json",
+    "build-worker": "tsc -p ./worker/tsconfig.json",
     "karma": "karma",
     "mocha": "mocha",
-    "tsc": "tsc"
+    "test": "bash ./unit_tests/test.sh",
+    "tsc": "echo 'Run ./build.sh instead' && false",
```

Comment:
At the least, please revert this `tsc` line change; we can always put that sort of warning in the README.md if it becomes an issue for folks.  Also, it'd fail on first download, since there's no default version of worker.js to include.  (This is my one absolute sticking point for this PR.)

There's a decent chance the `tsc` may prove useful for test resource development; we may want to compile some testing utility functions against LMLayer code.